plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -350,15 +377,33 @@ def get_isolation_state(self):
                 port_state.update(Constants.STATE_ISOLATED, Constants.ISSUE_OONOC)
                 self.ports_states[port] = port_state
 
+    def start_telemetry_session(self):
+        self.logger.info("Starting telemetry session")
+        response = self.ufm_client.start_dynamic_session(Constants.PDR_DYNAMIC_NAME, self.telemetry_counters, self.t_isolate)
+        if response and response.status_code == http.HTTPStatus.ACCEPTED:
+            port = str(response.content)
+        else:
+            self.logger.error(f"Failed to start dynamic session: {response}")
```

Comment:
what happens in case of start session failed - will the plugin retry to start the session later on?